xchange-core/src/main/java/org/knowm/xchange/dto/marketdata/CandleStick.java

```diff
@@ -0,0 +1,123 @@
+package org.knowm.xchange.dto.marketdata;
+
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+/**
+ * Data object representing a CandleStick
+ */
+@JsonDeserialize(builder = CandleStick.Builder.class)
+public class CandleStick {
+
+    private final Date date;
+    private final BigDecimal open;
+    private final BigDecimal high;
+    private final BigDecimal low;
+    private final BigDecimal close;
+    private final BigDecimal volume;
+    private final BigDecimal currencyVolume;
+
+    public CandleStick(Date date,BigDecimal open, BigDecimal high, BigDecimal low, BigDecimal close, BigDecimal volume, BigDecimal currencyVolume) {
```

Comment:
code formatting is not according to our conventions

please build using maven - the autoformatter will fix this

Reply:
Formatted.